ext/soap/php_encoding.c

```diff
@@ -3041,6 +3042,7 @@ static xmlNodePtr to_xml_list(encodeTypePtr enc, zval *data, int style, xmlNodeP
 	xmlAddChild(parent, ret);
 	FIND_ZVAL_NULL(data, ret, style);
 
+	/* Literals are uniqued and can't refer to other references via attributes. */
```

Comment:
Typo?
```suggestion
	/* Literals are unique and can't refer to other references via attributes. */
```

Reply:
I thought uniqued was a verb meaning "making something unique" but it seems that I'm wrong in that regard